openpype/hosts/maya/plugins/publish/extract_thumbnail.py

```diff
@@ -110,8 +134,9 @@ def process(self, instance):
             preset["viewport2_options"]["transparencyAlgorithm"] = transparency
 
         # Isolate view is requested by having objects in the set besides a
-        # camera.
-        if preset.pop("isolate_view", False) and instance.data.get("isolate"):
+        # camera. If there is only 1 member it'll be the camera because we
+        # validate to have 1 camera only.
+        if instance.data["isolate"] and len(instance.data["setMembers"]) > 1:
             preset["isolate"] = instance.data["setMembers"]
```

Comment:
This logic change should be highlighted in the PR I'd say - this changes how the isolate view logic works. Previously it'd have listened to the `isolate_view` setting regardless, now it silently ignores it if it thinks it didn't find anything other than the camera.

Not too bothered by it myself, but it does make the isolate view option less explicit.

Reply:
This is technically a bug fix cause the code was not working as originally intended. The original [comment](https://github.com/ynput/OpenPype/blob/0019ff6f7da8f8bb21d7b4b88d3bb84f24bfdc90/openpype/hosts/maya/plugins/publish/extract_playblast.py#L123-L126), indicates this behaviour which I've elaborated on.